src/main/java/uk/gov/ons/ctp/response/sample/service/PartySvcClientService.java

```diff
@@ -58,11 +58,12 @@ public PartyDTO postParty(final PartyCreationRequestDTO partyCreationRequestDTO)
       try {
         party = objectMapper.readValue(responseBody, PartyDTO.class);
       } catch (IOException e) {
-        String msg = String.format("cause = %s - message = %s", e.getCause(), e.getMessage());
-        log.error(msg);
+        log.with("cause", e.getCause())
+            .with("message", e.getMessage())
+            .error("Unable to read party response");
```

Comment:
I done the same but `.error("Unable to read party response", e)` makes more sense as @NickGrantONS suggested